torch/csrc/autograd/python_function.cpp

```diff
@@ -63,6 +70,12 @@ static PyObject* _allocate_grad_output(output_info_type& info, AutoGPU& gpu_guar
 
 namespace torch { namespace autograd {
 
+/**
+ * Legacy implementation of apply, which is invoked during the backwards pass
+ * when backward is NOT implemented as a static method.  A lot of user-code
+ * defines this method as a regular method, so this function is still important,
+ * but PyTorch should be moving away from this.
```

Comment:
PyTorch is moving away from this, so it's best to not include it in the comment, as it will be outdated (hopefully) soon